XMetaL/Display/MiscellaneousDocument.css

```diff
@@ -162,10 +166,17 @@ Language,
     { margin-left: 80pt;
       text-indent: -72.5pt;}
 
+
+DateLastModified           { margin-left: 17pt; }
+/* TranslationOf              { text-indent: -138pt; }  */
```

Comment:
Why the addition of a rule which is commented out?

Reply:
Because it had been added, tested, found to be unnecessary, and then commented out.